tasks/SmoothieChef.tex

```diff
@@ -0,0 +1,29 @@
+\section{Smoothie Chef [Party Host]}
+The home owner will teach the robot to make a smoothie which guests at the party really like!
+
+\subsection{Focus}
+Learning from demonstration.
+
+\subsection{Main Goal}
+The home owner will teach the robot using demonstrations how to make a smoothie. The robot is then to repeat the task. A designated team member will show the home owner how to perform the demonstration. To avoid liquids and blenders, we will have the robot place the fruit corresponding to the desired smoothie into a bowl.
+
+\noindent\textbf{Reward:} 2500pts (500 points per ingredient).
+
+
+\subsection{Setup}
+\begin{itemize}[nosep]
+	\item \textbf{Kitchen Counter:} Place plastic fruit onto the counter in the kitchen.
+	\item \textbf{Ingredient Bowl:} Place bowl into the kitchen.
+\end{itemize}
+
+\subsection{Referee instructions}
+The referee needs to
+\begin{itemize}
+	\item Witness the team member demonstration.
```

Comment:
I'm not quite sure about this. So the team member demonstrates to the operater how to do it and then the operator to the robot? Is the first part included in the test's time?
One of the general approaches in @home is that only the robot is allowed to instruct the operator. I do not see why the robot could not explain how it needs things to be demonstrated. 
Otherwise we also had teams give like a handout for certain tests which contained for example how to address the robot in the restaurant test.